### PR TITLE
MCS-1138: Add Scene Option to Toggle All Lights Off or On at Specific…

### PR DESCRIPTION
**Pull request by alexcraneunity**
This introduces the option to add light-toggling into the JSON scene files, which causes the lights in the scene to be disabled at the given stepBegin, and re-enabled at the given stepEnd.  Multiple pairs of begin and end can be added to toggle the lights multiple times in a scene. 

The holes.json changes were included as an example for your own testing, but I will remove them before the PR is merged.

### CODE REVIEW

File: unity/Assets/Addressables/MCS/Scenes/holes.json [hunk 1]
```diff
@@ -9,6 +9,15 @@
...
     },
+    "toggleLights": [
+        {
+            "stepBegin": 3,
+            "stepEnd": 6
+        }, {
+            "stepBegin": 8,
+            "stepEnd": 9
+        }
+    ],
```
Comment (brianpippin) [inline unity/Assets/Addressables/MCS/Scenes/holes.json:20]: All the code looks good, just curious why the toggleLights is in a holes.json file?  If we keep example scenes in here, should this be in a generic scene called toggleLights.json so people know where to look to find an example?

Comment (alexcraneunity) [reply unity/Assets/Addressables/MCS/Scenes/holes.json]: Yes it was left is as an example, I was planning on taking it out before the review was merged. Good suggestion - I went ahead and removed it, and created a toggle_lights_example json scene file.

**Review by jaudick — APPROVED** [review]

**Comment by ThomasSchellenbergNextCentury** [comment]
@alexcraneunity @brianpippin Unfortunately I had to deal with some problems generating the eval data earlier today and didn't get to finish looking at this PR until after it had been merged, but there are some significant issues that should get resolved ASAP.

- The lights need to be re-enabled in `MCSMain.ChangeCurrentScene`. Right now, if they turn off, and then you load a new scene using the same MCS controller (Unity instance), they're still off when the new scene loads. This is definitely something the TA1s will encounter when training!
- There's no PR for the corresponding Python (MCS) branch, which is required for this feature to work.

I've made [MCS-1409](https://nextcentury.atlassian.net/browse/MCS-1409) to address these issues.

**Comment by alexcraneunity** [comment]
Thank you @ThomasSchellenbergNextCentury  - I will create the missing PR for MCS-1138 in the MCS repo, and will work on resolving these issues. Should we revert the merge commit for ai2thor?

**Comment by ThomasSchellenbergNextCentury** [comment]
@alexcraneunity 

> Should we revert the merge commit for ai2thor? 

I thought about doing that, but I don't think we need to. We're not planning on releasing what's on `development` soon. I just don't want us to forget about fixing it :)

**Comment by alexcraneunity** [comment]
Follow-up code review: https://github.com/NextCenturyCorporation/ai2thor/pull/256

**Pull request merged**